README.md

```diff
@@ -144,8 +135,8 @@ tasks.check {
 <summary>Groovy</summary>
 
 ```groovy
-check {
-    dependsOn "installKotlinterPrePushHook"
+tasks.named('check') {
+    dependsOn 'installKotlinterPrePushHook'
```

Comment:
I committed to `'` for Groovy scripts, as that seemed to be the convention here. I personally prefer `"` as they work both in Groovy and in Kotlin DSL scripts. It feels more practical, but didn't want to change too much.

If the old single quote isn't a hard requirement, I can easily update them all.